connect/cli/plugins/product/sync/translations.py

```diff
@@ -23,6 +25,7 @@ def __init__(self, client, silent, stats):
         self._mstats = stats['Translations']
 
     def sync(self):
+        rows_data = defaultdict(dict)
         for row_idx, data in self._iterate_rows():
             self._set_process_description(f'Processing Product translation {data.translation_id}')
             if data.action == '-':
```

Comment:
In the task it says: **Note 2: primary translation should be skipped for actions**, so maybe the ` _validate_row` method should not validate an action for a primary translation and add it to the list of errors, it should happend outside and be skipped like:
```
if data.action == '-' or data.is_primary == 'Yes':
    self._mstats.skipped()
    continue
```
In other words the suggestion is not to be an error of the `sync` process, wdyt?

Reply:
You are right, I totally missed this, and in fact the primary translation can be updated. I've checked in Connect and this would raise a 400 Bad request, so it should definitely be skipped.
Thanks for pointing this!

Reply:
Fixed.
Also updated the export command, now the action for primary translation has only the '-' option, to make more clear to the user that it cannot be modified.